accelerator/apis.h

```diff
@@ -42,6 +42,24 @@ void apis_logger_init();
  */
 int apis_logger_release();
 
+/**
+ * Initialize lock
+ *
+ * @return
+ * - zero on success
+ * - SC_CONF_LOCK_INIT on error
+ */
+status_t apis_lock_init();
```

Comment:
I wonder the necessity of lock initialization API. Can you just regard it as internal functions?

Reply:
We still need a `apis_init` function to trigger the lock initialization.

Reply:
Let's consolidate all initialization process in single-entry function calls. That is, no matter how lock initializations are implemented, don't expose its behavior in public functions. We can encapsulate in existing init function.